src/icon-buttons/use-add-button-styling.component.js

```diff
@@ -0,0 +1,29 @@
+import React, {useEffect, useState} from 'react'
+
+export default function useAddButtonStyling (props) {
+  useEffect(() => {
+    let styleElement = document.getElementById('bandicoot-button-styling')
+    if (!styleElement) {
+      styleElement = document.createElement('style')
+      styleElement.setAttribute('id', 'bandicoot-button-styling')
+      styleElement.textContent = css
+      document.head.appendChild(styleElement)
+      return () => styleElement.parentNode.removeChild(styleElement)
+    }
+  }, [])
+}
+
+const css = `
+  .bandicoot-button-styling {
+    background-color: transparent;
+    border: none;
+    border-radius: 4px;
+  }
+  .bandicoot-button-styling:hover {
+    transition: background-color .25s ease-in-out;
+    background-color: rgba(0, 0, 0, 0.05);
+  }
+  .bandicoot-button-styling svg {
+    fill: currentColor;
+  }
+`
```

Comment:
how do we let the consumer style the `active-control-button`? I think we have to leave it to them to get a style on the page for it. There is too much potential variety to accept props